### PR TITLE
Add nice logos for downloads

### PR DESCRIPTION
**Pull request by licaon-kter**
Hope the Github/F-Droid/XDA logo sources don't break anything.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,12 +12,20 @@ Blocking access to the internet can help:
 NetGuard is possibly the first free and open source no-root firewall for Android.
 
 Downloads:
+--------
+
+<a href="https://github.com/M66B/NetGuard/releases"><img src="https://assets-cdn.github.com/images/modules/logos_page/GitHub-Logo.png" height="32" ></a> (stable)
+
+[![Google Play](http://developer.android.com/images/brand/en_generic_rgb_wo_60.png)](https://play.google.com/store/apps/details?id=eu.faircode.netguard&referrer=utm_source%3Dgithub) (stable)
```
Comment (M66B) [inline README.md]: The width and height (see screenshots) need to be set, else the anchors don't position correctly on all browsers.

Comment (M66B) [inline README.md]: Better remove this: *&referrer=utm_source%3Dgithub*

**Comment by M66B** [comment]
I like this, but I have the image links won't break in the near future. I tend to support my project for a long time (XPrivacy almost 2,5 years now).

File: README.md (hunk 1 above)
Comment (licaon-kter) [inline README.md]: I hoped it's useful ( point 2. ): https://developers.google.com/analytics/devguides/collection/android/v4/campaigns#google-play-campaigns

Comment (M66B) [inline README.md]: I don't like to track people in any way.

**Comment by licaon-kter** [comment]
Better?

**Comment by M66B** [comment]
Still missing the height for *en_generic_rgb_wo_60.png*.

Why not use: https://f-droid.org/wp-content/uploads/2014/04/cropped-header_resp1.png

**Comment by licaon-kter** [comment]
That's 60 already, I'll need to change the way it's linked though.

**Comment by licaon-kter** [comment]
Now?

**Pull request merged**